android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -727,7 +729,15 @@ public void handleJsonEvent(final JsonReadEvent jsonReadEvent) {
 
                     OpenEventApp.postEventOnUIThread(new MicrolocationDownloadEvent(true));
                     break;
-                } default:
+                }case ConstantStrings.SESSION_TYPES: {
+                    List<SessionType> sessionTypes = objectMapper.readValue(json, new TypeReference<List<SessionType>>() {});
```

Comment:
will do that. thanks for pointing out.
can you test the app? because it crashes after downloading the stuff and it is not because of this as per the logs. 
just confirm this. Thanks.